tests/code-splitting/subtests/dynamic-import-preloading/rollup/src/profile.js

```diff
@@ -0,0 +1,8 @@
+import { exclaim } from './exclaim.js';
```

Comment:
This file shouldn't be here

Reply:
I don't understand it.

Reply:
It's not part of the test case described in the readme. It's unneeded to demonstrate the issue. It can be removed and also removed from rollup config